Introduction.md

```diff
@@ -0,0 +1,44 @@
+# Why Another Validator?
+Firedancer will be solving some of the problems which solana as a network and community is trying to Solve.
+1. **Compute Fee and gas fee estimatation** -> Firedancers new and better blockpackaging algorithm does a lot better predicting the amount of Compute units used in a specific trnsaction making it easier for validators to choose which ones to include in the next block.When taken it makes 3 times less mistakes in predicting the CUs used. ![image](https://user-images.githubusercontent.com/88841339/224479879-466202fb-d085-4c93-beb6-22d6c3885a44.png)
+
+2. **Robustness**:When the validators are unable to reach consesnus and verify a transaction leading to a fork and downtime. One of the biggest memes floating around about Solana is its downtime, though it was a design tradeoff for it being the most performant blockchain, downtimes lead to a lot of different issues like liquidity, validator restarts etc which are mostly casued by small bugs in the updates in the core client code. 
+When there are 2 client implementations and one of them faces a bug, the nodes running the bug free code will continue processing transactions to which the other nodes could sync to later. 
+		
+ # How?
+ Firedancer has 3 high level Components which are modular and can be replaced with the components of Old Solana validator making it like different  components being stitched together while also writing a better documentation of the original validator client
+- Components->
+	- 	Packet Ingress
+	- 	Runtime
+	- 	Consensus
+
+ ![image](https://user-images.githubusercontent.com/88841339/224098753-4a62136a-aad7-4343-89b0-f8c0fd45a990.png)
+
+- # Why C?
+	- When Building something like Firedancer which could potentially be run on thousands of machines, the code has to be efficient. C allows were tight integration with the hardware and make it allow developers a lot of flexibility in the ability to manage compute resources. [Here Josh Siegel](https://www.youtube.com/watch?v=9efhIs37hVI) one of the core builders of Firedancer explain in depth and practical demo on "why C"?
+- # Block propagation in Firedancer
+	- ![Image](https://pbs.twimg.com/media/FmSWXByXwAAD1oH?format=jpg&name=4096x4096)
+
+---
+	
+### Wanna Learn more about firedancer? make sure to check out the [announcement](https://jumpcrypto.com/jump-vs-the-speed-of-light/) and the breakpoint talks.
+- Breakpoint Talks by the legend KFB -> [General overview](https://www.youtube.com/watch?v=Dh6Yn2Odyr4)
+
+	- [Deep Dive and first demo ](https://www.youtube.com/watch?v=YF-7duYCK54)
+	
+	- [Second Demo during the solana community call ](https://www.youtube.com/watch?v=zFS7MY4spBE)
+			[The slides]
+	- Link to the resentations ->
+		- [Deep Dive](https://jump-assets.storage.googleapis.com/2022-11-06-bowers-et-al-deep-dive-final-embed-public.pdf)
+		- [Intros](https://jump-assets.storage.googleapis.com/2022-11-05-bowers-et-al-hold-me-closer-frankendancer-breakpoint-2022-final-embed-public.pdf)
```

Comment:
I will reformat these but have to run offline.

Reply:
It looked pretty good when opened in obsidian, wish github had a better markdown viewer